src/contracts/misc/sGhoSteward.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {AccessControl} from 'src/contracts/dependencies/openzeppelin-contracts/contracts/access/AccessControl.sol';
+import {SafeCast} from 'src/contracts/dependencies/openzeppelin-contracts/contracts/utils/math/SafeCast.sol';
+
+import {IsGhoSteward} from './interfaces/IsGhoSteward.sol';
+import {IsGHO} from '../sgho/interfaces/IsGho.sol';
+
+/**
+ * @title sGhoSteward
+ * @author BGD Labs
+ * @notice Helper contract for managing rate and supply cap parameters for sGho.
+ */
+contract sGhoSteward is AccessControl, IsGhoSteward {
+  using SafeCast for uint256;
+
+  /// @notice Current rate parameters
+  RateConfig public rateConfig;
+
+  IsGHO public immutable sGHO;
```

Comment:
In the other contracts in the repo these are prefixed and then params on functions don't have prefix/suffix from what i've seen